integrations/http/http.go

```diff
@@ -171,15 +201,17 @@ func setBody(req *http.Request, rawBody string, formBody map[string]string, json
 		// This is required when using ioutil.NopCloser method for the request body
 		// (see ShouldSendChunkedRequestBody() in the library mentioned above).
 		req.ContentLength = int64(len(rawBody))
-		return nil
-	}
 
-	// JSON body.
-	if jsonBody.IsValid() && !jsonBody.IsNothing() {
-		if formBody != nil {
-			return errMutuallyExclusive
+		// Set the Content-Type header only if it's not already set.
+		if req.Header.Get(contentTypeHeader) == "" {
+			req.Header.Set(contentTypeHeader, "text/plain") // or "application/octet-stream"
```

Comment:
do we need the comment?

Reply:
i wonder if we need this header at all. since it's a raw body, it's the user's responsibility. you can add body_type `text` if you want to.

Reply:
httpbin won't respond if the content type isn't set. We define raw_body to be string and not bytes, so I think it's nicer to set content type and allow user to override it if he wants.
But I removed the comment.